app/jobs/search_keyword_job.rb

```diff
@@ -4,11 +4,26 @@ class SearchKeywordJob < ApplicationJob
   queue_as :default
 
   def perform(keyword_id)
-    keyword = Keyword.find(keyword_id)
-    html = GoogleSearchService.new(keyword: keyword.keyword).call
-    utf_8_html = html.force_encoding('iso8859-1').encode('utf-8')
-    keyword.add_html(utf_8_html)
-  rescue ActiveRecord::RecordNotFound, GoogleSearch::Errors::SearchKeywordError, ActiveRecord::StatementInvalid
-    keyword.update_status(:failed)
+    @keyword = Keyword.find(keyword_id)
+    begin
+      html = GoogleSearchService.new(keyword: keyword.keyword).call
```

Comment:
```suggestion
      html_response = GoogleSearchService.new(keyword: keyword.keyword).call
```

Reply:
Sorry for late reply! 
I fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)